content/learn/beginner/datatypes-1.md

```diff
@@ -0,0 +1,131 @@
++++
+title = "Data Types 1"
+weight = 220
++++
+
+In this chapter, we will go over the basic data types that are available in the bot's template actions. These data types
+are used to store various types of data, such as numbers, strings, and more. We will also go over how to use variables
```

Comment:
"We will overview how to store data in variables and the basic data types available, which include numbers, strings, and more."

I think presenting data types before data is debatable pedagogy. Data first seems more reasonable to me.

Reply:
Certainly agreed, I'm not too sure what I was thinking when I did that.